akka-actor-tests/src/test/scala/akka/io/UdpConnectedIntegrationSpec.scala

```diff
@@ -94,6 +95,41 @@ class UdpConnectedIntegrationSpec extends AkkaSpec("""
       expectMsgType[Udp.Received].data should ===(data1)
     }
 
+    // #26903
+    "be able to send and receive when server goes away (and comes back)" in {
+      val addresses = temporaryServerAddresses(2, udp = true)
+      val serverAddress = addresses(0)
+      val clientAddress = addresses(1)
+      val server = bindUdp(serverAddress, testActor)
+      val data1 = ByteString("To infinity and beyond!")
+
+      val clientCommander = connectUdp(Some(clientAddress), serverAddress, testActor)
+      clientCommander ! UdpConnected.Send(data1)
+
+      expectMsg(Udp.Received(data1, clientAddress))
+
+      server ! Udp.Unbind
+      expectMsg(Udp.Unbound)
+      Thread.sleep(1000) // if it stops that takes a bit of time, give it that time
+
+      // bug was that the commander would fail on next read/write
+      clientCommander ! UdpConnected.Send(ByteString("data to trigger fail"), 1)
+      expectMsg(1)
```

Comment:
Kind of weird to have those acks in UDP... but OK I guess.

Reply:
It's only an ack that the udp connection actor enqueued the byte string to the os buffers.